docker/Dockerfile

```diff
@@ -61,7 +61,7 @@ RUN curl -fsSL -v -o ~/miniconda.sh -O  https://repo.anaconda.com/miniconda/Mini
         weasyprint && \
     /opt/conda/bin/conda clean -ya && \
     # Fix GLIBCXX version issue
-    rm -f /opt/conda/lib/libstdc++.so.6
+    cp /opt/conda/lib/libstdc++.so.6 /lib/x86_64-linux-gnu/libstdc++.so.6
```

Comment:
Do you mean you can't use that `cp` method in latest test?

Reply:
Yes, seems not work, failed to build `text`

Reply:
Reproduced. Because conda installed libstdc++6 version is 6.0.29 now, ubuntu 20.04 is 6.0.28 by default, but ubuntu 22.04 is 6.0.30. Reverted this change. We need to use ubuntu 22.04 as base image now.
cc: @WeizhuoZhang-intel